spring-batch-docs/asciidoc/retry.adoc

```diff
@@ -62,18 +62,18 @@ public interface RetryCallback<T> {
 }
 ----
 
-The callback is executed and if it fails (by throwing an
-    `Exception`), it will be retried until either it is
-    successful, or the implementation decides to abort. There are a number of
+The callback runs and, if it fails (by throwing an
+    `Exception`), it is retried until either it is
+    successful or the implementation aborts. There are a number of
     overloaded `execute` methods in the
-    `RetryOperations` interface dealing with various use
-    cases for recovery when all retry attempts are exhausted, and also with
+    `RetryOperations` interface. Those methods deal with various use
+    cases for recovery when all retry attempts are exhausted and deal with
```

Comment:
`and handles retry state` instead of `and deal with`.  Thoughts?

Reply:
Six of one, half dozen of the other, I think, so feel free to change it.